frontends/mit-open/src/pages/SearchPage/SearchPage.tsx

```diff
@@ -222,47 +232,95 @@ const TABS: TabConfig[] = [
 ]
 const ALL_RESOURCE_TABS = TABS.map((t) => t.resource_type)
 
+/**
+ * Augment the facet buckets for `groupKey` with active values that have no
+ * results.
+ */
+const includeActiveZerosInBuckets = (
+  groupKey: string,
+  aggregations: LearningResourceSearchResponse["metadata"]["aggregations"],
+  params: LRSearchRequest,
+) => {
```

Comment:
@abeglova Maybe the `FacetDisplay` component should handle this? OCW does seem to add-in active facets with zero count, but I don't think it's in course-search-utils currently?

Reply:
Sounds good. I'd like to do that separately from https://github.com/mitodl/course-search-utils/pull/78, but I can do it before merging this PR.

https://github.com/mitodl/course-search-utils/pull/78 is big enough already, and I'd like replace the Enzyme-based facet tests if I'm going to be adding more there.